lib/login/login.app.js

```diff
@@ -12,38 +32,38 @@ function loadModule() {
+        .config(['ConfigUtilsProvider', '$cookiesProvider', '$uibTooltipProvider', '$logProvider', function (ConfigUtilsProvider, $cookiesProvider, $uibTooltipProvider, $logProvider) {
+            $cookiesProvider.defaults.path = '/';
+            $uibTooltipProvider.options({ appendToBody: true });
+            $logProvider.debugEnabled(ConfigUtilsProvider.$get().getConfigJSON().debug === true);
+        }])
 
-            .run(['AlertsService', 'messageMap', 'Session', 'ERMrest', 'UiUtils', 'UriUtils',
-                function (AlertsService, messageMap, Session, ERMrest, UiUtils, UriUtils) {
-                    try {
-                        var subId = Session.subscribeOnChange(function () {
-                            Session.unsubscribeOnChange(subId);
-                            var session = Session.getSessionValue();
-                            if (!session && Session.showPreviousSessionAlert())
-                                AlertsService.addAlert(messageMap.previousSession.message, 'warning', Session.createPromptExpirationToken);
-                        });
-                        UriUtils.setLocationChangeHandling();
-                        UiUtils.setBootstrapDropdownButtonBehavior();
-                    } catch (exception) {
-                        throw exception;
-                    }
-            }]);
+        .run(['AlertsService', 'messageMap', 'Session', 'ERMrest', 'UiUtils', 'UriUtils',
+        function (AlertsService, messageMap, Session, ERMrest, UiUtils, UriUtils) {
+            try {
+                var subId = Session.subscribeOnChange(function () {
+                    Session.unsubscribeOnChange(subId);
+                    var session = Session.getSessionValue();
+                    if (!session && Session.showPreviousSessionAlert())
+                    AlertsService.addAlert(messageMap.previousSession.message, 'warning', Session.createPromptExpirationToken);
+                });
+                UriUtils.setLocationChangeHandling();
+                UiUtils.setBootstrapDropdownButtonBehavior();
+            } catch (exception) {
+                throw exception;
+            }
+        }]);
 
-            /**
-             * Manually initialize the ng-app
-             * Otherwise angular throws an error if ng-app is used in the HTML template as angular tries to load the module before all the dependencies have finished loading
-             */
-            angular.element(document).ready(function(){
-                angular.bootstrap(document.getElementsByTagName("login")[0],["chaise.loginapp"]);
-            });
+        /**
+        * Manually initialize the ng-app
+        * Otherwise angular throws an error if ng-app is used in the HTML template as angular tries to load the module before all the dependencies have finished loading
+        */
+        angular.element(document).ready(function(){
+            angular.bootstrap(document.getElementsByTagName("head")[0],["chaise.configure-login"]);
```

Comment:
Was there a reason that you changes this to head? This might cause the issue that Cris found. We don't want to turn the whole HTML page to Angularjs and just the login/navbar apps.

Reply:
I added a `configure-login` and `configure-navbar` angular-app that can't be on the same element or child element as another angular app (login app is attached tot he `login` tag element and navabr app is attached to the `navbar` tag element).

I didn't want to introduce another element that would force cris to have to change html templates for the static pages.